sources/packages/web/src/components/common/AwardTable.vue

```diff
@@ -57,7 +59,10 @@ export default defineComponent({
       }
       // If the award in not defined at all it means that the award is not eligible and it was not
       // part of the disbursement calculations output.
-      return awardValue ?? "(Not eligible)";
+      if (awardValue === undefined) {
+        return "(Not eligible)";
+      }
+      return getFormattedAwardValue(awardValue);
```

Comment:
As we know that the dynamic award value with key as award name is definitely a number, I would say we can force cast it.  like this. 

```ts
const awardValue = props.awardDetails[
        `${props.identifier}${awardType.toLowerCase()}`
      ] as number;
```
It will allow us to set the type of award value as number.